### ISSUE TITLE
Suggestion: choose page type

### ISSUE THREAD

**Issue by nacer80** [quran/quran_android#1492]
السلام عليكم ورحمة الله وبركاته 
@ahmedre and @benomaire 
Mostly people are using the Classic Madani Mushaf; I think because in the beginning the application starts download the first page type (Classic Madan Mushaf).
So I suggest to put message apears in the beginning when we install or update the application.

![Screenshot_20201108-085618_Word](https://user-images.githubusercontent.com/60948528/98458960-f98d3200-21a6-11eb-9fe0-e20b48def40d.jpg)
![Screenshot_20201108-085233_Word](https://user-images.githubusercontent.com/60948528/98458970-0742b780-21a7-11eb-8a5c-fabe81a956aa.jpg)
![Uploading Screenshot_20201108-085128_Word.jpg…]()

And when we click ok the app appears the page types so we can choose each Mushaf page we need.
وجزاكم الله خيرا

**Comment by benomaire** [comment]
I'd prefer making the 1441 version the default. Some features:

- Colorful
- Ayah symbol is never at the start of the line
- Surah header is never at the end of the page
- New Tanween shape

This should be easier for us and the users. May be we can keep a screenshot showing the old print on play store as a hint that other versions exist.

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله 
I agree with you, the version 1441 is very good Mash'Allah, but I think if you let the user chooses between the versions is better.

**Comment by ahmedre** [comment]
وعليكم السلام ورحمة الله وبركاته
agree with either of these - needs some thought.
one thing that historically held me back is we don't have word by word info for the 1441 and 1439 mus7afs - this prevents us from doing the word by word hover feature with the meaning of each word that i wanted to add (like brother Mohamed added to our iOS app).

definitely agree and should think about this more though.
والسلام عليكم

**Comment by AbdullahM0hamed** [comment]
> word by word hover feature with the meaning of each word
I've seen this on the Quran.com site, but is that a feature of the app? And how do we go about creating the database for this?

**Comment by ahmedre** [comment]
It's there on the iOS app, not on the Android one because the Android one supports page types for which we don't have the word by word coordinates.

**Comment by AbdullahM0hamed** [comment]
Looking at the [Qur'an-iOS](https://github.com/quran/quran-ios), the best I could find was [words.db](https://github.com/quran/quran-ios/blob/master/Quran/Resources/words.db), which has a single word_position column that contains an integer - how do they determine the location of a word on a page?

**Comment by ahmedre** [comment]
there's another database that has the location of each word.

**Comment by nacer80** [comment]
> السلام عليكم ورحمة الله وبركاته 
> @ahmedre and @benomaire 
> Mostly people are using the Classic Madani Mushaf; I think because in the beginning the application starts downloading the first page type (Classic Madani Mushaf).
> So I suggest to put a message apears in the beginning when we install or update the application.
> 
> 
> ![Screenshot_20201108-085618_Word](https://user-images.githubusercontent.com/60948528/98458960-f98d3200-21a6-11eb-9fe0-e20b48def40d.jpg)
> 
> 
> And when we click ok the app goes to page types in the setting so we can choose each Mushaf page we need.
> وجزاكم الله خيرا 
> 

@benomaire 
Now in last update of Golden quran app they added the option of choosing page type when we open the app after update 

Screen shot 
![Screenshot_20210430-101541_Golden Quran.jpg](https://user-images.githubusercontent.com/60948528/116676136-866ab880-a99e-11eb-9169-aefe85de8de1.jpg)

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله وبركاته 

It very important to have this option, while installing or updating the app, because many users didn't know other versions of Mushaf in the app.
وجزاكم الله خيرا

**Comment by RohanRahman-s** [comment]
I also agree with this post. Should allow users to choose on first launch, might as well mention techinal benefits/drawbacks of each one if every feature isn't fully supported by every mushaf. 

Also adding the naskh(indopak) font to the offical app and fixing its borders, removing lines in between, side comments of waqf. Would be nice. It has a clunky dual page mode as well.

**Comment by ahmedre** [comment]
agree with doing this in sha' Allah.

speaking of IndoPak, I am spending a bunch of time thinking about it these days to figure out how to replace the current images with the King Fahd Quran Complex images. My motivation is that I got an email I got saying "the side comment isn't there telling me there's waqf lazem" here (even though the text properly has the stop symbol). Someone else once complained that the "manzil" number on one of the pages is wrong, and someone else sent an email saying there's a fat7a missing on one of the words - these types of emails are the motivation to swap the pages since I lost all trust in these pages.

maybe you can help me by answering some questions about it: mostly about the lines in between the lines and the side comments - should these be toggle-able? I would love to just remove them, but am not sure if I can (especially the side comments) due to the emails I mentioned above. Appreciate any opinions on this.

**Comment by nacer80** [comment]
> وعليكم السلام ورحمة الله وبركاته
> agree with either of these - needs some thought.
> one thing that historically held me back is we don't have word by word info for the 1441 and 1439 mus7afs - this prevents us from doing the word by word hover feature with the meaning of each word that i wanted to add (like brother Mohamed added to our iOS app).
> 
> definitely agree and should think about this more though.
> والسلام عليكم ورحمة الله وبركاته 
The word by word info with meaning is  available in Surah app provided by Tafsir center.
You can ask Sheikh Abderhmen Elshehri about it.
amshehri@tafsir.net

**Comment by RohanRahman-s** [comment]
> agree with doing this in sha' Allah.
> 
> speaking of IndoPak, I am spending a bunch of time thinking about it these days to figure out how to replace the current images with the King Fahd Quran Complex images. My motivation is that I got an email I got saying "the side comment isn't there telling me there's waqf lazem" here (even though the text properly has the stop symbol). Someone else once complained that the "manzil" number on one of the pages is wrong, and someone else sent an email saying there's a fat7a missing on one of the words - these types of emails are the motivation to swap the pages since I lost all trust in these pages.
> 
> maybe you can help me by answering some questions about it: mostly about the lines in between the lines and the side comments - should these be toggle-able? I would love to just remove them, but am not sure if I can (especially the side comments) due to the emails I mentioned above. Appreciate any opinions on this.

I'm just somewhat suprised that some people need the waqf lazim on the side to know which mark it is. Couldn't there be a first page that shows what the marks mean so that one learns them first? Perhaps even translate it. Lots of websites explaon that the م is waqf lazeem. ط waqf mutlaq etc. Most mushaf have an explanation at the end or the beginning of it. Perhaps providing that may undermine the need of the side text. And if said explanation is in Urdu only or Arabic Only, providing a translation depending on interface language may be beneficial. 

It is worth remembering that not everyone who uses these apps are tech savy and would know how to toggle the marks on or off. I personally don't see the purpose of the side marks if we could provide a first page with the Stop marks/wuquf and their meanings in the chosen language.


Regarding the Manzil.... I am not sure which issue it is, but I could look it up on one of my 'subcontinental' mushafs and check, not all mushafs have the exact same manzil and hizb numbering. 

Although the missing fatha case is something more serious worth checking out. Just compare the mentioned error with its madani or shemerly counterpart. I have seen a flood of people downloading the warsh and qaloon apps who claim that there are mistakes all over due to ignorance of the qiraat, is this one of those cases?
